### ISSUE TITLE
Bandwidth Graph Anomaly in Brave

### ISSUE THREAD

**Issue by lidel** [ipfs-shipyard/ipfs-webui#1012]
- Web UI version: `QmfQkD8pBSBCBxWEwFSu4XaDVSWK6bjnNuaWZjMyQbyDub` (v2.4.4)
- Brave versions: 0.61, 0.63.35 :anger:  (issue seen on screenshots below)
- Firefox version: 66 :angel:  (works ok)
- Google Chrome version: 73.0  :angel: (works ok)

### Problem

While  working on [js-ipfs in Brave](https://github.com/ipfs-shipyard/ipfs-companion/issues/664) I noticed embedded HTTP API exposed by js-ipfs produced buggy graph (note "s" on the left axis on screens below). 
 
### js-ipfs 0.35.0-rc.5

Sample graph from Brave:
> ![2019-04-11--22-19-51](https://user-images.githubusercontent.com/157609/55990708-b0ae8800-5ca8-11e9-9ae1-a90501f59200.png)
Sample js-ipfs response for `/api/v0/stats/bw`:
> ```json
> {"TotalIn":"31848851","TotalOut":"1440904","RateIn":"6.158484647860612","RateOut":"5.028479187074011"}
> ```

### go-ipfs 0.4.19

Sample graph from Brave:
> ![2019-04-11--22-47-06](https://user-images.githubusercontent.com/157609/55992191-4697e200-5cac-11e9-8a8c-b06bcea8cc43.png)
> 
> Sample go-ipfs response for `/api/v0/stats/bw`:
> ```json
> {"TotalIn":183758816,"TotalOut":355038782,"RateIn":5376.543583100996,"RateOut":9682.160239299323}
> ```
---- 

### Reproducibility

Did some testing and this issue occurs in regular [Brave Beta](https://brave.com/download-beta/) using vanilla Web UI. Interestingly, the problem does not show on Google Chrome Beta, looking at the same Web UI+API produces normal graph. I tested this on two linux machines. 

Is anyone else using Brave and can confirm this is a real issue and not something on my end? 
(The ask is to open Web UI in Brave via menu item in IPFS Companion)

**Comment by olizilla** [comment]
Yep, I can recreate on Brave beta `Version 0.64.60 Chromium: 74.0.3729.91 (Official Build) beta (64-bit)` on macOS

<img width="1552" alt="Screenshot 2019-04-23 at 16 23 01" src="https://user-images.githubusercontent.com/58871/56594249-6ac8bc80-65e4-11e9-99e5-18f624335550.png">
